docs/smt_rmt_suma.md

```diff
@@ -1,51 +1,64 @@
+|Registration of CentOS 8 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of CentOS 7 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of CentOS 6 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Oracle Linux 8 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Oracle Linux 7 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Oracle Linux 6 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of openSUSE Leap 15 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Ubuntu 20.04 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Ubuntu 18.04 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Ubuntu 16.04 systems|:x:|:x:|:heavy_check_mark:|
+|Support for migrating SLE 12 systems to 15|:warning:<sup>[1](#partial-migration)</sup>|:heavy_check_mark:|:heavy_check_mark:|
+|Support for migrating SLE 15 SPx systems to 15 SPx+1|:warning:<sup>[1](#partial-migration)</sup>|:heavy_check_mark:|:heavy_check_mark:|
+|Staging repositories|:heavy_check_mark:|:x:<sup>[2](#staging)</sup>|:heavy_check_mark:|
+|Air gap sync/mirroring for secure environments|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|NTLM Proxy support|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Custom repositories|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|YaST installation wizard|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|YaST management wizard|:heavy_check_mark:|:x:|:x:|
+|Client management|:heavy_check_mark:|:x:|:heavy_check_mark:|
+|Red Hat 7 and earlier support ([Expanded Support](https://www.suse.com/products/expandedsupport/))|:heavy_check_mark:|:x:|:heavy_check_mark:|
+|Red Hat 8 support ([Expanded Support](https://www.suse.com/products/expandedsupport/))|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Files deduplication|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Transfer registration data to SCC|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Reporting|:heavy_check_mark:|:x:|:heavy_check_mark:|
+|Custom TLS certificates for web-server|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Clean up data from repositories that are not used any longer|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Bash completion|:x:|:heavy_check_mark:|:x:|
+|Available on [openSUSE Leap 15](https://github.com/SUSE/rmt/blob/master/docs/installation.md#installation-on-opensuse-leap-15)|:x:|:heavy_check_mark:<sup>[3](#self-support)</sup>|:x:|
+|Option to [run as container](https://github.com/SUSE/rmt/blob/master/README.md#development-setup---docker-compose)|:x:|:heavy_check_mark:<sup>[3](#self-support)</sup>|:x:|
+|Easy development setup + [contribution guide](https://github.com/SUSE/rmt/blob/master/docs/CONTRIBUTING.md)|:x:|:heavy_check_mark:|:x:|
+|100% test [coverage](https://coveralls.io/github/SUSE/rmt?branch=master)|:x:|:heavy_check_mark:|:x:|
+|[Plugin functionality](https://github.com/SUSE/rmt/blob/master/docs/PLUGINS.md)|:x:|:heavy_check_mark:|:x:|
```

Comment:
SUSE Manager has plugins for some things, e. g. virtual host managers and cluster managers